app/styles/common.scss

```diff
@@ -1,11 +1,15 @@
+@use '~bootstrap-sass/assets/stylesheets/bootstrap';
+@use 'variables';
+@use 'typography';
```

Comment:
> Are you using `@use 'typography';` in this file? I might have missed it if you are.

`@extend .sans-serif;` and similar is a reference to `typography.scss`.

> I guess the typography file is just added classes, but then why was it removed from style.scss?

Before `style.scss` `@import`ed everything once into the global scope. `@import` is deprecated now and we have to `@use`. `@use` only makes the variables and styles from that module available in the current file. So now we have to to `@use` `variables` and `typography` in every file that references them. There's also no reason to have the import in `style.scss` anymore because it doesn't actually use any of the variables and styles that `variables.scss` or `typography.scss` define. Hopefully that makes sense.